### PR TITLE
[FIX] install/maintain: update versions in "Installing Odoo"

### PR DESCRIPTION
**Pull request by StanvanHoorn**
Correct version from 13.0 to 15.0

### CODE REVIEW

File: content/administration/install/install.rst [hunk 1]
```diff
@@ -107,7 +107,7 @@ Linux
 Debian/Ubuntu
 '''''''''''''
 
-Odoo 13.0 'deb' package currently supports `Debian Buster`_, `Ubuntu 18.04`_ or above.
+Odoo 15.0 'deb' package currently supports `Debian Buster`_, `Ubuntu 18.04`_ or above.
```
Comment (d-fence) [inline content/administration/install/install.rst:110]:
```suggestion
Odoo 15.0 'deb' package currently supports `Debian 11 (Bullseye)`_, `Ubuntu 20.04 (Focal)`_ or above.
```

File: content/administration/install/install.rst [hunk 2]
```diff
@@ -180,7 +180,7 @@ and automatically start the server.
 Fedora
 ''''''
 
-Odoo 13.0 'rpm' package supports Fedora 30.
+Odoo 15.0 'rpm' package supports Fedora 30.
```
Comment (d-fence) [inline content/administration/install/install.rst:183]:
```suggestion
Odoo 15.0 'rpm' package supports Fedora 34.
```

**Review by d-fence — CHANGES_REQUESTED** [review]
Nice catch of those poor forgotten documentation lines.
Can you change your commit message so that it conforms to the [guidelines](https://www.odoo.com/documentation/15.0/developer/misc/other/guidelines.html#tag-and-module-name)

It should looks like
`[FIX] install: update versions`

**Comment by StanvanHoorn** [comment]
Done!

**Comment by Xavier-Do** [comment]
Also, you may need to rebase your branch if you want to push again, it is a little outdated.

**Comment by Xavier-Do** [comment]
Build is red:


```
/data/build/documentation/content/administration/install/install.rst:110: WARNING: Unknown target name: "debian 11 (bullseye)".
/data/build/documentation/content/administration/install/install.rst:110: WARNING: Unknown target name: "ubuntu 20.04 (focal)".
```

File: content/administration/install/install.rst [hunk 3]
```diff
@@ -107,7 +107,7 @@ Linux
 Debian/Ubuntu
 '''''''''''''
 
-Odoo 13.0 'deb' package currently supports `Debian Buster`_, `Ubuntu 18.04`_ or above.
+Odoo 15.0 'deb' package currently supports `Debian 11 (Bullseye)`_, `Ubuntu 20.04 (Focal)`_ or above.
```
Comment (d-fence) [inline content/administration/install/install.rst:110]: Sorry, I forgot to mention that you also have to change the references accordingly in lines 755 and 759 hence the warnings.

**Review by d-fence — CHANGES_REQUESTED** [review]

**Comment by jcs-odoo** [comment]
Having a quick look at this.
Can you squash both commits and have a commit message such as the following, please? 
[FIX] install/maintain: update versions in "Installing Odoo"

Here is the [documentation's commit message template](https://github.com/odoo/documentation/blob/15.0/commit_template.txt).

Well... now that I see it, basically what @d-fence  wrote :D 

Thanks!

**Comment by jcs-odoo** [comment]
I believe this would solve the issue #1304

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate=jcs-odoo

**Pull request closed without merging**